### DISCUSSION TITLE
Https everywhere

### DISCUSSION THREAD

**Discussion by mimmosowlo** [bromite/bromite#1044] [Ideas]
Is it possible to enable https everywhere (or something like it) in bromite?

**Comment by chromer030** [comment]
Google Chrome 90 will automatically turn the half-finished URL into HTTPs pages. It may be possible to implement this option in Bromite also.

**Reply by oldphones68** [reply]
What is `half-finished URL` actually? Would you please explain? @chromer030

**Reply by chromer030** [reply]
When user types URL without protocol strings, it's called half-finished by Google.
For example bing.com is half-finished and https://bing.com is full-finished URL.

**Comment by oldphones68** [comment]
@csagan5 It could be a giant step forward in terms of security. Couldn't it?
